### ISSUE TITLE
Found 34 files excluded from sound null safety in webdev/

### ISSUE THREAD

**Issue by devoncarew** [dart-lang/webdev#1640]
Found 34 files excluded from sound null safety in webdev/:

```
webdev/bin/webdev.dart:5:// @dart = 2.9
webdev/lib/src/command/build_command.dart:5:// @dart = 2.9
webdev/lib/src/command/configuration.dart:5:// @dart = 2.9
webdev/lib/src/command/daemon_command.dart:5:// @dart = 2.9
webdev/lib/src/command/serve_command.dart:5:// @dart = 2.9
webdev/lib/src/command/shared.dart:5:// @dart = 2.9
webdev/lib/src/daemon/app_domain.dart:5:// @dart = 2.9
webdev/lib/src/daemon/daemon.dart:5:// @dart = 2.9
webdev/lib/src/daemon/daemon_domain.dart:5:// @dart = 2.9
webdev/lib/src/daemon/domain.dart:5:// @dart = 2.9
webdev/lib/src/daemon/utilites.dart:5:// @dart = 2.9
webdev/lib/src/daemon_client.dart:5:// @dart = 2.9
webdev/lib/src/logging.dart:5:// @dart = 2.9
webdev/lib/src/pubspec.dart:5:// @dart = 2.9
webdev/lib/src/serve/chrome.dart:5:// @dart = 2.9
webdev/lib/src/serve/dev_workflow.dart:5:// @dart = 2.9
webdev/lib/src/serve/handlers/favicon_handler.dart:5:// @dart = 2.9
webdev/lib/src/serve/server_manager.dart:5:// @dart = 2.9
webdev/lib/src/serve/utils.dart:5:// @dart = 2.9
webdev/lib/src/serve/webdev_server.dart:5:// @dart = 2.9
webdev/lib/src/util.dart:5:// @dart = 2.9
webdev/lib/src/webdev_command_runner.dart:5:// @dart = 2.9
webdev/test/build/min_sdk_test.dart:5:// @dart = 2.9
webdev/test/chrome_test.dart:5:// @dart = 2.9
webdev/test/configuration_test.dart:5:// @dart = 2.9
webdev/test/daemon/app_domain_test.dart:5:// @dart = 2.9
webdev/test/daemon/daemon_domain_test.dart:5:// @dart = 2.9
webdev/test/daemon/launch_app_test.dart:5:// @dart = 2.9
webdev/test/daemon/utils.dart:5:// @dart = 2.9
webdev/test/e2e_test.dart:5:// @dart = 2.9
webdev/test/integration_test.dart:5:// @dart = 2.9
webdev/test/readme_test.dart:5:// @dart = 2.9
webdev/test/test_utils.dart:5:// @dart = 2.9
webdev/test/utils_test.dart:5:// @dart = 2.9
```

See https://github.com/dart-lang/sdk/issues/49169 for more context.

**Comment by annagrin** [comment]
Duplicate of https://github.com/dart-lang/webdev/issues/1327

**Comment by devoncarew** [comment]
@annagrin - this issue is for webdev; it looks like #1327 just covers package:dwds. Is that issue also meant to track migrating webdev to null safety?

**Comment by annagrin** [comment]
@devoncarew you are right, let's keep this one for webdev only

**Comment by mit-mit** [comment]
@annagrin will you also be migrating this one? if so, got an eta?

**Comment by annagrin** [comment]
@elliette is the current owner of the task.

**Comment by mit-mit** [comment]
Got it. @elliette do you have an estimate?

**Comment by elliette** [comment]
This is on our Q4 OKRs - I expect it will be done by EOY / beginning of Q1 2023. Thanks!

**Comment by vsmenon** [comment]
@elliette - we'd like to get this done by EOY.  Do you think that's still on track?

**Comment by elliette** [comment]
Yes, it's on track! I finished migrating the non-test directories yesterday, so only the tests are left.

**Comment by vsmenon** [comment]
@khyati82 @itsjustkevin - for tracking

**Comment by a-siva** [comment]
Are we going to do a release of the webdev package with the new null safe changes ?

**Comment by elliette** [comment]
Yes, I will! Planning on doing that this week after some manual sanity-check testing

**Comment by elliette** [comment]
Just published the null-safe version `2.7.12`
